plugins/utils/which-key.nix

```diff
@@ -14,7 +14,7 @@ with lib; {
     package = helpers.mkPackageOption "whick-key-nvim" pkgs.vimPlugins.which-key-nvim;
 
     registrations = mkOption {
-      type = with types; attrsOf str;
+      type = with types; attrsOf (oneOf [str (listOf str) (attrsOf str)]);
```

Comment:
> Actually, I would even go for `attrsOf anything` so that it is more flexible.

Yes! Changing the `type` to `attrsOf anything` would allow bindings with raw lua code, which would be closer to the native experience of which-key.